corehq/apps/userreports/tasks.py

```diff
@@ -46,7 +46,9 @@ def _get_db(doc_type):
 
 
 # This is intentionally not using magic to introspect the class from the name, though it could
+from corehq.apps.locations.models import Location
 _DOC_TYPE_MAPPING = {
     'XFormInstance': XFormInstance,
     'CommCareCase': CommCareCase,
+    'Location': Location
```

Comment:
Just stumbled across this PR while looking in to similar functionality.  How difficult would it be to not use `corehq.apps.locations.models.Location` here?  We've removed all couch views that used the `Location` doc type, and I'd very much like to get rid of the model entirely.  In my mind, the only reason it's still around is because we haven't yet invested the effort required to fully remove it.